packages/arb-ts/integration_test/arb-bridge.test.ts

```diff
@@ -385,14 +386,172 @@ describe('ERC20', () => {
         testWalletL2Balance.add(tokenWithdrawAmount).eq(tokenDepositAmmount)
     ).to.be.true
   })
-  // it('withdraw events queries works as expected',async ()=>{
-  //   const l1EventData = await bridge.getL2ToL1EventData(l1TestWallet.address)
-  //   const withdrawTokenData = await bridge.getTokenWithdrawEventData(
-  //     l1TestWallet.address
-  //   )
-  //   console.log(l1EventData.length, withdrawTokenData.length)
-  //   expect(l1EventData.length).to.equal(withdrawTokenData.length)
-  // })
+})
+
+describe('CustomToken', () => {
+  let l1CustomToken: TestCustomTokenL1
+  let l2CustomToken: TestArbCustomToken
+  if (!existantCustomTokenL1 && !existantCustomTokenL2) {
+    it('sets up a new custom token, L1 and L2', async () => {
```

Comment:
This looks like a test setup, should we move it to a `before` hook? Then we can evaluate the `if/else` inside the before hook instead of using it to determine which test cases we run.